tests/features/communities/tallinn/permissions.feature

```diff
@@ -0,0 +1,65 @@
+@api
+Feature:
+  - As a user, author of a Tallinn Report node, I want to be able to edit the
+    node that I own but I cannot edit other user's Tallinn Reports.
+  - As a moderator, when editing a Tallinn Report node, I want to be able
+    edit the author of the node.
+
+  Scenario: Test permissions on Tallinn Reports.
+    Given users:
+      | Username  | Roles     |
+      | vasile    |           |
+      | dominique |           |
+      | chef      | moderator |
+      | gheorghe  |           |
+    And collection:
+      | title | Kind of Tallinn |
+      | state | validated       |
+    And the following collection user memberships:
+      | collection      | user      |
+      | Kind of Tallinn | vasile    |
+      | Kind of Tallinn | dominique |
+    And tallinn_report content:
+      | title   | author    | collection      |
+      | Romania | gheorghe  | Kind of Tallinn |
+      | France  | dominique | Kind of Tallinn |
+
+    # A user outside of Tallinn collection is not able to edit any report.
+    Given I am logged in as gheorghe
+    When I visit the tallinn_report content "Romania" edit screen
+    Then I should get a 403 HTTP response
+    When I visit the tallinn_report content "France" edit screen
+    Then I should get a 403 HTTP response
+
+    # A moderator is able to change any report's author.
+    Given I am logged in as chef
+    When I visit the tallinn_report content "Romania" edit screen
```

Comment:
:D :D I would also have used "Greece" most probably if I were to write that test :P